lib/api/v1/reports.js

```diff
@@ -46,14 +46,27 @@ exports.methods = {
     };
 
     try {
-      const locationID = await knex('locations').insert(fullLocation, ['id']);
-      const fullReport = {
-        category: report.category,
-        description: report.description,
-        image: report.image,
-        email: report.email,
-        location_id: locationID[0].id
-      };
+      const existingLocationID = await knex('locations').where('lat', `${location.lat}`).andWhere('long', `${location.long}`);
+      let fullReport = {}
+      if (existingLocationID.length > 0) {
```

Comment:
Adding this conditional logic allows us to not create duplicate locations on the reports endpoint